apps/watch/src/components/VideoContentPage/VideoContentPage.tsx

```diff
@@ -37,6 +37,17 @@ export function VideoContentPage(): ReactElement {
   const [openShare, setOpenShare] = useState(false)
   const [openDownload, setOpenDownload] = useState(false)
 
+  const ogSlug =
+    container !== undefined
+      ? `${container.slug}.html/${
+          variant?.slug === undefined ? '' : variant.slug
+        }.html`
+      : `${
+          variant?.slug === undefined
+            ? ''
+            : `${variant.slug.split('/')[0]}.html/${variant.slug.split('/')[1]}`
+        }.html`
+
```

Comment:
```suggestion
  let ogSlug = ''

  if (
    container?.slug != null &&
    label != null &&
    ![VideoLabel.collection, VideoLabel.series].includes(label)
  ) {
    ogSlug += `/${container.slug}.html/${variant?.slug ?? ''}.html`
  } else {
    const [videoId, languageId] = (variant?.slug ?? '').split('/')
    ogSlug = `/${videoId}.html/${languageId}.html`
  }
```

This is taken from the VideoCard component. Perhaps we ought to make this a sharable function between the two so that link generation remains consistent?

Reply:
Good idea.